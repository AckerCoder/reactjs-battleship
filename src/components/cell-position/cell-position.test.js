import React from "react";
import { shallow } from "enzyme";
import CellPosition from "./cell-position.component";

it("Expect to Match Snapshot", () => {
  expect(shallow(<CellPosition />)).toMatchSnapshot();
});

it("Expect to render CellComponent", () => {
  const mockPlayerName = "Player Name";
  const mockPosition = 1245;
  const mockScore = 12346;

  expect(
    shallow(
      <CellPosition
        playerName={mockPlayerName}
        position={mockPosition}
        score={mockScore}
      />
    ).length
  ).toEqual(1);
});
