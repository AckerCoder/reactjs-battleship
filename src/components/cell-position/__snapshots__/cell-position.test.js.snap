// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`Expect to Match Snapshot 1`] = `
<div
  className="cell-position"
>
  <span
    className="data"
  />
  <span
    className="data"
  />
  <span
    className="data"
  />
</div>
`;
